app/controllers/users_controller.rb

```diff
@@ -0,0 +1,6 @@
+class UsersController < ApplicationController
+  def show
+    @user = User.find(params[:id])
+    @parties = @user.parties
```

Comment:
I would be inclined to have it so we, as much as possible instantiate a single instance in a controller action, so would just have `@user` here.

In the view, we could call @user.parties